.github/workflows/synchronizing.yaml

```diff
@@ -96,8 +96,8 @@ jobs:
 
       - name: apply changes to the files
         run: |
-          if echo "${{ github.event.head_commit.message }}" | grep -q "#"; then
-            branch_name=$(echo "${{ github.event.head_commit.message }}" | grep -o '[0-9]\+')
+          if echo "${{ github.event.head_commit.message }}" | grep -q '\(#\d*\)'; then
+            branch_name=$(echo "${{ github.event.head_commit.message }}" | grep -o '#\d*')
```

Comment:
'#\d+' ?

Reply:
+ doesnt work in this grep engine for some reason

Reply:
Is the intent here to set the branch name to e.g. "#123" ?